tasks/task06/Decorators.py

```diff
@@ -0,0 +1,39 @@
+import functools
+import time
+
+
+def measure_elapsed_time(func):
+    """Print the runtime of the decorated function"""
+    @functools.wraps(func)
+    def wrapper_timer(*args, **kwargs):
```

Comment:
```suggestion
    def wrapper(*args, **kwargs):
```
`wrapper` это такой же стандарт, как `args` и `kwargs`. Нет смысла так уточнять название обертки